src/components/atoms/ToggleGroup/ToggleGroup.tsx

```diff
@@ -9,18 +9,21 @@ import {
 
 export type ToggleGroupItem = {
   label: string;
-  value: string;
+  value: {
+    fileId: string;
+    order: 'asc' | 'desc';
+  };
 };
 
 export interface ToggleGroupProps {
   items?: ToggleGroupItem[];
-  selectedValue?: string;
-  onClick?: (value: string) => void;
+  selectedValue?: { fileId: string; order: 'asc' | 'desc' };
+  onClick?: (value: { fileId: string; order: 'asc' | 'desc' }) => void;
```

Comment:
지금도 좋지만, 반복되는 공통 타입을 분리하는건 어떨까요?

```suggestion
  export type ToggleGroupValue = {
  fileId: string;
  order: 'asc' | 'desc';
};

export type ToggleGroupItem = {
  label: string;
  value: ToggleGroupValue;
};

export interface ToggleGroupProps {
  items?: ToggleGroupItem[];
  selectedValue?: ToggleGroupValue;
  onClick?: (value: ToggleGroupValue) => void;
}
```

Reply:
[b817aef](https://github.com/JECT-Study/PICK-O-Client/pull/224/commits/b817aef44b7a1f0ef9355f4b3f7e499c240f63ed) 에서 수정하였고 각 컴포넌트에 [8937321](https://github.com/JECT-Study/PICK-O-Client/pull/224/commits/8937321e231ee0d74a72c6ae8a0d126f7d56519f) 에서  반영하였습니다~